src/templates/zh-CHT/base.html

```diff
@@ -0,0 +1,188 @@
+{% extends "base/2019/base.html" %}
```

Comment:
This should be in `src/templates/zh-CHT/2019/` folder and not directly in the `src/templates/zh-CHT/` folder. Can you move it?

Reply:
sure.